solana/programs/example-native-token-transfers/src/instructions/transfer.rs

```diff
@@ -51,6 +66,15 @@ pub struct Transfer<'info> {
     #[account(mut)]
     pub outbox_rate_limit: Account<'info, OutboxRateLimit>,
 
+    #[account(
+        mut,
+        address = config.custody
+    )]
+    /// Tokens are always transferred to the custody account first regardless of
+    /// the mode.
+    /// For an explanation, see the note in [`transfer_burn`].
```

Comment:
or `transfer_lock`, I suppose

Reply:
this is phrased a little awkwardly -- my intention here was to say something like
"For an explanation of why the custody account is needed even in burning mode, see the note in [`transfer_burn`]" as that's the less intuitive case (and indeed that's the only place where there is a note to this effect)